Changing JIT perf test names to fix how they are organized on BenchView

jorive opened the pull request:
The names used on the BenchView reporting site are taken from the fully qualified names of the methods running the benchmarks (marked with the xunit performance attribute [Benchmark]). Before these changes, a lot of the classes/methods were in the global namespace, and all the tests would show up in a long list. Now, after these changes, tests are grouped.

In addition, I append the all the BenchView results into a single file and upload it once.

dnfclas commented:
Hi __@jorive__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.dotnetfoundation.org.
    </span>

TTYL, DNFBOT;

dnfclas commented:
__@jorive__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, DNFBOT;

dpodder commented on tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs:
```diff
@@ -24,13 +24,15 @@
 this class, so pointer comparison can be used for value comparison.
 */
 
+namespace V8
```
Is this left as `V8` instead of `V8.DeltaBlue` on purpose?

dpodder replied:
@jorive That's fine then to leave as-is; thanks for clarifying!

dpodder commented:
@jorive Thanks for the changes! Looks good now. Only issue left is to fix the test issue, I think. @DrewScoggins PTAL as well.

jorive commented:
@dotnet-bot test Windows_NT perf

dpodder commented:
@dotnet-bot test Windows_NT perf

jorive commented:
@dotnet-bot test Windows_NT CoreCLR Perf Tests

brianrob commented:
@dotnet-bot test Windows_NT CoreCLR Perf Tests

mmitche commented:
@dotnet-bot help

mmitche commented:
@dotnet-bot test Windows_NT perf

jorive commented:
@mmitche, @dpodder, @DrewScoggins I updated the perf.groovy to make sure we are running with the latest BenchView tools.

DrewScoggins commented:
In the latest tools did you make a change to make sure that the file exists before we append?  It looks like that is why the PR runs were failing.

jorive commented:
@DrewScoggins yes, I updated the tools (fixed the append option), but the nuget command did not pull the latest scripts. It was caused by the -ExcludeVersion option (if folder exist, then skip). It was not downloading the latest version.

This time, I moved where we download to, and verify that if the folder exist, then remove it and download it again.

DrewScoggins commented:
LGTM

dpodder commented:
@dotnet-bot test Windows_NT perf

mmitche commented:
@dotnet-bot test Windows_NT perf

jorive commented:
@dotnet-bot test Windows_NT perf

jorive commented:
@dotnet-bot test Windows_NT perf

DrewScoggins commented:
All the tests have passed, this LGTM.
